Update 1-variables.md (#4)
dragonfly91/java-class: basics/2-loops.md

}
```

# Count（）

> For each of the loops in the [Iterations](#iterations) question, print the number of iterations via code.

Reviewer: ```suggestion
For each of the loops in the previous question, print the number of iterations via code.
```

このリンクがもう起動しないので、無くしておけです！

Sum of even numbers from 2 to 10 is 30
```

# Sum Squares（2乗の合計）

Given an input n, write a program to calculate the sum of squares of integers up to n: 1<sup>2</sup> + 2<sup>2</sup> + 3<sup>2</sup> + ... + n<sup>2</sup>.

> nの数が与えられたとき、1^1+2^2+3^3+... + n^nの合計を計算するプログラムを書きましょう。

Reviewer: ```suggestion
nの数が与えられたとき、1<sup>2</sup> + 2<sup>2</sup> + 3<sup>2</sup> + ... + n<sup>2</sup>の合計を計算するプログラムを書きましょう。
```

Sum of squares of numbers from 1 to 10 is 385
```

# Product 1 to N（1からNまでのかけ算の合計）

> Write a program called Product1ToN to compute the product of integers from 1 to 10 (i.e., 1x2x3x...x10), as an integer. Take note that it is the same as factorial of N.

Reviewer: ```suggestion
Write a program called Product1ToN to compute the product of integers from 1 to N (i.e., 1x2x3x...xN), as an integer. Take note that it is the same as factorial of N.
```

Expected Input:

> 1から10までの整数の積（すなわち1x2x3x...x10）を整数で計算するProduct1ToNというプログラムを作成しましょう。Nの階乗と同じであることに注意しましょう。

Reviewer: ```suggestion
1からNまでの整数の積（すなわち1x2x3x...xN）を整数で計算するProduct1ToNというプログラムを作成しましょう。Nの階乗と同じであることに注意しましょう。
```

Expected Output:

できるもの：

```bash
Maximum: 6
```

Now modify the program to make it work for 3 numbers

> 今度は、3つの数字に対して動作するようにプログラムを修正します。

Reviewer: ```suggestion
今度は、3つの数字に対して動作するようにプログラムを修正してみよう。
```

Expected Output:

できるもの：

```bash
Maximum: 10
```

Now modify the program to make it work for n numbers, where n is also an input

> 今度は、n個の数字に対して動作するようにプログラムを修正します。

Reviewer: ```suggestion
今度は、n個の数字に対して動作するようにプログラムを修正してみよう。
```

Expected Output:

できるもの：

```bash
Maximum: 6
```

Now modify the program to make it print the minimum as well.

> 今度は、最小値も表示されるようにプログラムを修正します。

Reviewer: ```suggestion
今度は、最小値も表示されるようにプログラムを修正してみよう。
```

Expected Output:

できるもの：

```bash
Maximum: 6
Minimum: 2
```

> # Harmonic Sum（ハーモニックサム？）

Reviewer: ```suggestion
# Harmonic Sum（ハーモニック足し算）
```
